manifest.yaml

```diff
@@ -8,7 +8,7 @@ containers:
         mountPath: /data
 
   - name: guestbook
-    image: google/guestbook-python-redis
+    image: gcr.io/your-project-id/guestbook-python-redis
```

Comment:
The README instructions have a way for a user to run the example without building it.  As such, we should have a public image someplace and refer to that with manifest.yaml.

Or we should instruct users to always build it and modify manifest.yaml.  In that case, I'd have this be `manifest.yaml.in` or something and either have them copy/modify it or have some `sed` magic in `start-containers.sh`.  Note you can detect the project the user has configured with: `$(gcloud config list project --format=text | cut -d ' ' -f 2)`